### PR TITLE
[Docathon][Fix note No.8-10]

### PR DESCRIPTION
**Pull request by KANGslay**
内容：
调整rst note标签缩进

修改：
docs/api/paddle/nn/BatchNorm1D_cn.rst   
docs/api/paddle/nn/BatchNorm2D_cn.rst 
docs/api/paddle/nn/BatchNorm3D_cn.rst

Issue: https://github.com/PaddlePaddle/docs/issues/7134
@Echo-Nie @sunzhongkai588

### CODE REVIEW

**Comment by luotao1** [comment]
@Echo-Nie 可以review

File: docs/api/paddle/nn/BatchNorm1D_cn.rst [hunk 1]
```diff
@@ -3,8 +3,8 @@
 BatchNorm1D
 -------------------------------
 
-.. py:class:: paddle.nn.BatchNorm1D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCL', use_global_stats=None, name=None)
-
+.. py:class::
+    paddle.nn.BatchNorm1D(num_features, momentum=0.9, epsilon=1e-05, weight_attr=None, bias_attr=None, data_format='NCL', use_global_stats=None, name=None)
```
Comment (luotao1) [inline docs/api/paddle/nn/BatchNorm1D_cn.rst:7]: 这几行为什么要修改？
